### ISSUE TITLE
Sync Error for with multiple nested namespaces

### ISSUE THREAD

**Issue by j-ales** [psychopy/psychopy#5408]
I have nested namespaces on Pavlovian.  These cause on error because the following line tries to assign more than 2 values incorrectly.

pavlovia.py line ~1330: nameSpace, projectName = path.split("/")

**Comment by peircej** [comment]
I don't follow. Do you mean you have a "/" within the project name? I didn't know that was even possible!
Let us know a bit more info how you've got this set up and we can try to work around it

thanks

**Comment by j-ales** [comment]
No.  It's not within the project name.  I have namespaces that contain namespaces.   
See for example the "st-andrews" namespace contains both individual projects and an "aleslab" namespace  that contains an its own set of projects.  See the following project which is nested down two levels: https://gitlab.pavlovia.org/st-andrews/aleslab/stepTrack2022

I've currently worked around it by avoiding the nested namespace.

**Comment by peircej** [comment]
@TEParsons this is solved in 2023.2 right?
